src/Data/Nat/Divisibility.agda

```diff
@@ -79,20 +119,20 @@ m%n≡0⇔n∣m m n = mk⇔ (m%n≡0⇒n∣m m n) (n∣m⇒m%n≡0 m n)
   divides (q * p) (sym (*-assoc q p _))
 
 ∣-antisym : Antisymmetric _≡_ _∣_
-∣-antisym {m}     {zero}  _ (divides-refl q) = *-zeroʳ q
-∣-antisym {zero}  {n}     (divides p eq) _   = sym (trans eq (*-comm p 0))
-∣-antisym {suc m} {suc n} p∣q           q∣p  = ≤-antisym (∣⇒≤ p∣q) (∣⇒≤ q∣p)
+∣-antisym {m}     {zero}   _  q∣p = equalityᵒ q∣p
+∣-antisym {zero}  {n}     p∣q  _  = sym (equalityᵒ p∣q)
+∣-antisym {suc m} {suc n} p∣q q∣p = ≤-antisym (∣⇒≤ p∣q) (∣⇒≤ q∣p)
 
 infix 4 _∣?_
 
 _∣?_ : Decidable _∣_
-zero  ∣? zero   = yes (divides-refl 0)
-zero  ∣? suc m  = no ((λ()) ∘′ ∣-antisym (divides-refl 0))
-suc n ∣? m      = Dec.map (m%n≡0⇔n∣m m (suc n)) (m % suc n ≟ 0)
+zero  ∣? zero   = Dec.yes (divides-refl 0)
```

Comment:
Is there a reason we're starting to qualify `yes` and `no`?

Reply:
Good question. It may be pure idiosyncrasy on my part. I'll look again.

Reply:
I suspect I had been thinking "the proof of `_∣?_ : Decidable _∣_` is the one place where decidability is appealed to, so let's localise use of `Relation.Nullary.Decidable` to this one place via a qualified import.

Easy enough to revert (but there is a secret agenda, about deprecating `yes` and `no` downstream, because they are too strict as *patterns*, while harmless as *definitions*/'smart constructors', so I guess I was laying a trail of breadcrumbs towards that cf issues #2058 #2059 and my various experiments around resolving those (plus #2055 and also #2199 ...)

Reply:
Okay, well that's a good point about their eventual deprecation. However, I think the trail is a little obtuse, and this change won't pro-actively help when and if those changes land. So can we revert it for now?

Reply:
Pushed.